sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client_helpers.py

```diff
@@ -32,7 +32,7 @@ def _parse_url(
         - except in the case of AzureSasCredential, where the conflicting SAS tokens will raise a ValueError.
         If using an instance of AzureNamedKeyCredential, "name" should be the storage account name, and "key"
         should be the storage account key.
-    :paramtype credential: Optional[Union[str, Dict[str, str], AzureNamedKeyCredential, AzureSasCredential, AsyncTokenCredential, TokenCredential]] # pylint: disable=line-too-long
+    :type credential: Optional[Union[str, Dict[str, str], AzureNamedKeyCredential, AzureSasCredential, AsyncTokenCredential, TokenCredential]] # pylint: disable=line-too-long
```

Comment:
As per discussion, moved the `credential` type to a newline to remove unnecessary `pylint: disable=line-too-long`, but this file is an exception since the union types will be too long to fit on one line